README.md

```diff
@@ -108,7 +109,7 @@ mvn install
 This will compile DDF and run all of the tests in the DDF source distribution. It usually takes some time for maven to download required dependencies in the first build.
 The distribution will be available under "distribution/ddf/target" directory.
 
-### How to build using multiple threads ###
+### How to build using multiple threads (Currently not usable on Windows) ###
```

Comment:
The `C` in `Currently` should be lower-case. I hate being _this_ precise when it's really not a big deal, but it would look more professional to be grammatically correct.